app/models/gobierto_plans/plan.rb

```diff
@@ -35,6 +35,14 @@ def nodes
       @nodes ||= Node.joins(categories: [:vocabulary]).where(terms: { vocabulary_id: categories_vocabulary&.id }).order(position: :asc, id: :asc)
     end
 
+    def have_indicators?
+      GobiertoCommon::CustomFieldRecord.includes(:custom_field).where(custom_fields: { instance_id: id, field_type: 10 }).count > 0
+    end
```

Comment:
You can reuse the `indicators` method for this
```suggestion
    def have_indicators?
      indicators.exists?
    end
```